main/integration-tests/README.md

```diff
@@ -2,9 +2,10 @@
 
 ## Intro
 
-The integration-tests package is for running API tests against all SWB APIs. 
-These tests can run against local and dev environments during development. 
+The integration-tests package is for running API tests against all SWB APIs.
```

Comment:
Can we say " The integration-tests package is **used** for running API tests against all SWB APIs."